### PR TITLE
Better light tiles

### PR DESCRIPTION
**Pull request by unid15**
- Hitting a light tile with a multitool now opens up a list of available colors, instead of cycling through them
- Allows setting custom colors for light tiles

### CODE REVIEW

**Comment by Unfit** [comment]
oh boy conflicts already

File: html/changelogs/Unfit_lighttiles.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   rscadd (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (PJB3005) [inline html/changelogs/Unfit_lighttiles.yml]: HERP DERP

**Comment by Unfit** [comment]
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: html/changelogs/Unfit_lighttiles.yml (hunk 1 above)
Comment (BestName) [inline html/changelogs/Unfit_lighttiles.yml]: :laughing:

**Pull request merged**